syntax = "proto2";

// A log print message
message DrawPlot {
  required uint32 plot_number = 1;
  required string data_label = 2;
  repeated uint32 data_x = 3 [packed=true];
  repeated float data_y = 4 [packed=true];
}